src/main/java/lotto/Rank.java

```diff
@@ -0,0 +1,49 @@
+package lotto;
+
+public enum Rank {
+    NONE(0, 0),
+    FIFTH(5000, 3),
+    FOURTH(50000, 4),
+    THIRD(1500000, 5),
+    SECOND(30000000, 5),
+    FIRST(2000000000, 6);
```

Comment:
숫자의 경우 3자리 단위로 `_`를 추가해줄 수 있는데요. 
이렇게 `_`를 넣어주면 숫자 값을 파악하기 수월할 것 같네요. 😉 

```suggestion
    NONE(0, 0),
    FIFTH(5_000, 3),
    FOURTH(50_000, 4),
    THIRD(1_500_000, 5),
    SECOND(30_000_000, 5),
    FIRST(2_000_000_000, 6);
```